src/main/java/de/doubleslash/keeptime/view/ProjectReport.java

```diff
@@ -41,11 +41,11 @@ public ProjectReport(final int size) {
 
    public void appendToWorkNotes(final String currentWorkNote) {
       this.numberOfNotes++;
+      this.sb.append(currentWorkNote.trim());
       if (!currentWorkNote.isEmpty()) {
-         if (this.numberOfNotes > 1) {
+         if (this.size>1) {
```

Comment:
we have tests which needs to be
a) checked if still working (pipeline fails, but not sure why. logs do not load currently)
b) extended with new behavior